### PR TITLE
add login page

### PR DESCRIPTION
**Pull request by sandmich**

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -31,7 +31,7 @@
         android:layout_marginStart="115dp"
         android:layout_marginEnd="115dp"
         android:layout_marginBottom="227dp"
-        android:backgroundTint="#2196F3"
+        android:backgroundTint="#8B9CBC"
```
Comment (stephchi0) [inline app/src/main/res/layout/activity_main.xml:34]: i think we should probably use the colors.xml for this kind of stuff. if the color doesn't exist there yet, you can add it

**Review by stephchi0 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (stephchi0) [reply app/src/main/res/layout/activity_main.xml:34]: same for the other stuff on the page

Comment (sandmich) [reply app/src/main/res/layout/activity_main.xml:34]: l8r problem merge rn

Comment (sandmich) [reply app/src/main/res/layout/activity_main.xml:34]: gimme green checkmark @stephchi0

Comment (stephchi0) [reply app/src/main/res/layout/activity_main.xml:34]: kk

**Review by stephchi0 — APPROVED** [review]

**Pull request merged**